engine/flink/management/src/it/resources/docker/Dockerfile

```diff
@@ -16,7 +16,7 @@ RUN chmod +x /docker-entrypoint.sh
 
 USER flink
 RUN mkdir -p /tmp/storage
-RUN mkdir -p /tmp/storage/1.4/blob/cache
+RUN mkdir -p /tmp/storage/1.7/blob/cache
```

Comment:
Wow it we have here old flink version? What does it cause? Please name this magic number so we will prevent future similar problems with it.

Reply:
I think it was not needed for a long time ;) I'll try to remove this line